src/migration/1759384808784-DropUserExperienceJobSkills.ts

```diff
@@ -0,0 +1,106 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
This is basically this file, just switched the `up` and `down` calls.

https://github.com/dailydotdev/daily-api/blob/main/src/migration/1753885628500-UserExperiences.ts